ocsp/responder.go

```diff
@@ -242,13 +244,18 @@ var hashToString = map[crypto.Hash]string{
 // strings of repeated '/' into a single '/', which will break the base64
 // encoding.
 func (rs Responder) ServeHTTP(response http.ResponseWriter, request *http.Request) {
+	ctx := request.Context()
 	le := logEvent{
 		IP:       request.RemoteAddr,
 		UA:       request.UserAgent(),
 		Method:   request.Method,
 		Path:     request.URL.Path,
 		Received: time.Now(),
 	}
+	beeline.AddField(ctx, "real_ip", request.RemoteAddr)
+	beeline.AddField(ctx, "method", request.Method)
+	beeline.AddField(ctx, "user_agent", request.UserAgent())
+	beeline.AddField(ctx, "path", request.URL.Path)
```

Comment:
Are these names because beeline has some standardized names for certain kinds of fields? Or is it because having a longer field name is basically free in structured logging?

Also, we have a `defer` here to trigger the debug log. I suspect this is just because I'm not yet versed in beeline, but: why don't we need an equivalent `defer` to log this stuff to honeycomb? And does honeycomb support a notion of debug level logs?

Reply:
These names are direct copies of the names currently set on the `logEvent` object. No other logic to them than that. All fields that we add to the span automatically have "app." prepended to the name we supply here, so there's no risk of collision with honeycomb-standardized fields.

The wrapper which creates the span also sends it. If we create our own span in application code (as opposed to just adding fields to an existing span), then yes we have to `defer span.Send()` that.

Honeycomb does not have a notion of debug-level logs; everything gets sent into the same dataset. The cost does not go up based on the number of fields you set on a span, so you can put all levels of information into a single span and simply filter what you want to view at query time.